cli-plugins/insight/cli-overview.hbs.md

```diff
@@ -6,11 +6,12 @@ Follow these steps to install, configure, and use your Tanzu Insight CLI plug-in
 
 >**Note:** Prior to using the CLI plug-in, you must install the Supply Chain Security Tools - Store, either as its own package, or as part of Tanzu Application Platform View profile.
 
-1. If the `insight` plug-in is not already installed, see [Install the Tanzu Insight plug-in](cli-installation.md)
+1. If the `insight` plug-in is not already installed, see [Install the Tanzu Insight plug-in](cli-installation.hbs.md)
 1. [Configure insight](cli-configuration.hbs.md)
 
 Once `tanzu insight` CLI plug-in is set up:
 
-1. [Add data](add-data.md)
-1. [Query data](query-data.md)
-1. [Triage vulnerabilities](triaging-vulnerabilities.md)
+1. [Add data](add-data.hbs.md)
+1. [Query data](query-data.hbs.md)
+1. [Triage vulnerabilities](triaging-vulnerabilities.hbs.md)
```

Comment:
In the TOC, you have the SBOM page above the Triage page. But in this list you flipped the order.